--- a/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/javac/plugins/errorprone/ErrorPronePlugin.java	1980-01-01 00:00:00.000000000 +0000
+++ b/src/java_tools/buildjar/java/com/google/devtools/build/buildjar/javac/plugins/errorprone/ErrorPronePlugin.java	2019-06-06 02:24:59.655111799 +0000
@@ -18,6 +18,7 @@
 import com.google.common.collect.ImmutableList;
 import com.google.devtools.build.buildjar.InvalidCommandLineException;
 import com.google.devtools.build.buildjar.javac.plugins.BlazeJavaCompilerPlugin;
+import com.google.errorprone.BaseErrorProneJavaCompiler;
 import com.google.errorprone.ErrorProneAnalyzer;
 import com.google.errorprone.ErrorProneError;
 import com.google.errorprone.ErrorProneOptions;
@@ -30,7 +31,7 @@
 import com.sun.tools.javac.comp.Env;
 import com.sun.tools.javac.main.JavaCompiler;
 import com.sun.tools.javac.util.Context;
-import com.sun.tools.javac.util.JavacMessages;
+//import com.sun.tools.javac.util.JavacMessages;
 import com.sun.tools.javac.util.Log;
 import java.util.Arrays;
 import java.util.List;
@@ -66,7 +67,8 @@
 
   /** Registers our message bundle. */
   public static void setupMessageBundle(Context context) {
-    JavacMessages.instance(context).add("com.google.errorprone.errors");
+  //  JavacMessages.instance(context).add("com.google.errorprone.errors");
+      BaseErrorProneJavaCompiler.setupMessageBundle(context);
   }
 
   @Override
