From fdf2d1b0eb728561ef65b57aeb0a7080575b6939 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 28 Aug 2019 05:58:52 +0000
Subject: [PATCH] support cross compiling on Yocto 2

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 cyber/croutine/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cyber/croutine/BUILD b/cyber/croutine/BUILD
index 1c660ae..cb0d09a 100644
--- a/cyber/croutine/BUILD
+++ b/cyber/croutine/BUILD
@@ -82,7 +82,7 @@ config_setting(
 
 config_setting(
     name = "arm_mode",
-    values = {"cpu": "arm"},
+    values = {"cpu": "armeabi"},
 )
 
 cpplint()
-- 
2.17.1

